docs/conf.py

```diff
@@ -145,7 +145,8 @@
 html_static_path = ['_static']
 
 def setup(app):
-    app.add_stylesheet("css/custom.css")
+    app.add_css_file("css/custom.css")
```

Comment:
The old name of `add_stylesheet` is deprecated. We have pinned sphinx to an older version in Trinity hence it's using the old name there. Just in case someone is wondering about this change.